tests/benchmark/spark/5_generate_results.sh

```diff
@@ -4,6 +4,9 @@ source PerfRun.conf
 directory=$outputLocation/$(date "+%Y.%m.%d-%H.%M.%S")_$dataSize$queries
 mkdir -p  $directory
 
+mv *.csv *.out $directory/
+cp PerfRun.conf $directory/
```

Comment:
Copy PerfRun.conf to preserve all the test settings. In case of Spark tests, copy spark-env.sh and spark-defaults.conf too.